preprocessors/utils.py

```
@@ -1,6 +1,7 @@
 import os
 import sys
 import time
+import hashlib
```

Ruff (0.8.2)

`hashlib` imported but unused

Remove unused import: `hashlib`

(F401)